imaging/ml/ml_codelab/README.md

```diff
@@ -21,14 +21,6 @@ Follow the steps listed [here](https://cloud.google.com/ai-platform/notebooks/do
 
 Click "OPEN JUPYTERLAB" to open JupyterLab UI.
 
-### Set-up environment
```

Comment:
Do we need `gcloud init` ? or does the default permissions work?

Reply:
seem like it works fine, with default service account